source/minio-cli/minio-mc/mc-rm.rst

```diff
@@ -23,6 +23,12 @@ To completely remove a bucket, use :mc:`mc rb` instead.
 Syntax
 ------
 
+.. |command| replace:: :mc-cmd:`mc rm`
```

Comment:
This is an example of a page that uses all of the version-specific options + replacements.

Each `.. |<string>|` is replaced by the string following `replace::`